Jenkinsfile

```diff
@@ -15,25 +15,29 @@
 
 def setupNodeAndTest(version) {
   node {
-    // Install CouchDB 1.6.1
-    docker.image('apache/couchdb:1.7.1').withRun('-p 5984:5984') {
-      // Install NVM
-      sh 'wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash'
-      // Unstash the built content
-      unstash name: 'built'
-      // Run tests using creds
-      withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'couchdb', usernameVariable: 'user', passwordVariable: 'pass']]) {
-        withEnv(["NVM_DIR=${env.HOME}/.nvm", "TAP_TIMEOUT=300"]) {
-          // Actions:
-          //  1. Load NVM
-          //  2. Install/use required Node.js version
-          //  3. Run tests
-          sh """
-            [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"
-            nvm install ${version}
-            nvm use ${version}
-            npm test && npm run unreliable-feed-test
-          """
+    // To get the docker sidecar run to default to docker-in-docker we must
+    // unset the DOCKER_HOST variable.
+    withEnv(["DOCKER_HOST="]){
+      // Install CouchDB 1.6.1
+      docker.image('apache/couchdb:latest').withRun('-p 5984:5984') {
```

Comment:
Should we fix this at `2.1.1` it's a shame there isn't a `2.x` or similar tag. I'm fine with `latest`, but we have to keep an eye out for CouchDB 3 breaking things. Maybe this fork won't need to live that long(!)

Reply:
There is actually:

![image](https://user-images.githubusercontent.com/6885295/37522892-a4b879f0-2925-11e8-8697-4cae7a948e58.png)

Source: https://hub.docker.com/_/couchdb/

Reply:
Good spot! I'll switch to `apache/couchdb:2`.

Reply:
nice, it's a shame they aren't listed [here](https://github.com/apache/couchdb-docker#available-tags)